src/SqlVirtualMachine/SqlVirtualMachine/SqlVirtualMachine/Cmdlet/NewAzureSqlVM.cs

```diff
@@ -0,0 +1,163 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Azure.Commands.SqlVirtualMachine.Common;
+using Microsoft.Azure.Commands.SqlVirtualMachine.SqlVirtualMachine.Model;
+using Microsoft.Rest.Azure;
+using static Microsoft.Azure.Commands.SqlVirtualMachine.Common.ParameterSet;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.SqlVirtualMachine.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.SqlVirtualMachine.SqlVirtualMachine.Cmdlet
+{
+    /// <summary>
+    /// Defines New-AzSqlVM cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlVM", DefaultParameterSetName = NameParameterList, ConfirmImpact = ConfirmImpact.Low, SupportsShouldProcess = true)]
```

Comment:
@rocchiele please remove the `ConfirmImpact` property from this attribute (`SupportsShouldProcess` should stay)